selene/src/roblox/base.toml

```diff
@@ -100,6 +105,14 @@ required = false
 type = "number"
 required = false
 
+[[BrickColor.new.args]]
+type = "string"
+required = false
+
+[[BrickColor.new.args]]
+type = { display = "Color3" }
+required = false
+

```

Comment:
I don't understand any of the BrickColor.new declarations at this point, this implies that there exists a valid...6 argument BrickColor.new usage.